book/installation/source.md

```diff
@@ -28,6 +28,7 @@ operating system:
 
 - **Ubuntu**: `apt-get install libclang-dev pkg-config build-essential`
 - **macOS**: `brew install llvm pkg-config`
+- **Windows**: `choco install make; choco install cmake --installargs 'ADD_CMAKE_TO_PATH=System'; choco install llvm; choco install protoc`
```

Comment:
We only need llvm, the rest are not needed

Reply:
Fixed